json-logic/schema.ts

```diff
@@ -0,0 +1,146 @@
+// LogicExpression refers to the JSONLogic standard.
+import { LogicExpression } from "./logic";
+
+/*
+* This module defines a schema for questionaire and result calulation logic.
+*/
+
+/**
+ * Type of a question, essentially defines the result type.
+ */
+export enum QuestionType {
+	Select = "Select",
+	Multiselect = "Multiselect",
+	Number = "Number",
```

Comment:
Integer or Decimal

Reply:
I'll introduce seperate QuestionTypes for integer and decimal, since it makes a difference in the UI.

Reply:
thx